alibi_detect/cd/tensorflow/classifier.py

```diff
@@ -144,7 +144,8 @@ def __init__(
         if isinstance(train_kwargs, dict):
             self.train_kwargs.update(train_kwargs)
 
-    def score(self, x: np.ndarray) -> Tuple[float, float, np.ndarray, np.ndarray]:  # type: ignore[override]
+    def score(self, x: np.ndarray) \
```

Comment:
`mypy` is failing because of the new line break causing the ignore to be on the next line